cmd/libsnap-confine-private/mount-opt.c

```diff
@@ -115,3 +117,122 @@ const char *sc_mount_opt2str(unsigned long flags)
 	}
 	return buf;
 }
+
+static void sc_grow_string(char **s, const char *extra)
```

Comment:
After discussing this on telegram we agreed to try a simpler constant size-buffer.